### ISSUE TITLE
Change text of item in scroll menu

### ISSUE THREAD

**Issue by EGAMAGZ** [jwlodek/py_cui#61]
This feature request is not related with any problem. It's just something that with the use of the library I notice and I think is missing

The feature I'm telling is the possibilty to change the text of a selected item in a Scroll Menu with a method. This method could receive the index of the item you want to change and the  new text, or only recieve the new text and take the value of the index that is inside the class.

Finally, I want to tell you that I really like the way you're developing this library, is really easy to use.

**Comment by jwlodek** [comment]
Sure I think this is a good idea. Will probably wait till the next release since the current next one has a lot of features and fixes included and I don't want to make too many changes at once. 

And glad you like it! My goal was to make it as easy/intuitive as possible.

**Comment by jupiterbjy** [comment]
I think we can simply use a public method dating back to a991d91 for this one.

https://github.com/jwlodek/py_cui/blob/b4938dd2c23a422496af7e32a33c2dbfcb348719/py_cui/ui.py#L863-L872

Since this returns direct reference to the list, we can just modify it.

Previously I used to clear, re-add lines within context manager so I can retain selected item and items currently visible on widget. That complex work could be done in one line with this method.

Or adding a convenient method would do.
```python
# in MenuImplementation

def set_item(self, idx: int, text: str):
    self.get_item_list()[idx] = text
```

**Comment by jwlodek** [comment]
A new function `set_selected_item` was added in the above PR, it will be available once `v0.1.3` is released (hopefully by the end of the month).
